src/main/resources/db_scripts/createDB.sql

```diff
@@ -0,0 +1,63 @@
+CREATE DATABASE quizowanie;
+USE quizowanie;
+
+CREATE TABLE question (
+	quest_id INT NOT NULL,
+	title VARCHAR(128) NOT NULL,
+	description VARCHAR(500),
+	PRIMARY KEY (quest_id)
+);
+
+CREATE TABLE answer (
+	ans_id INT NOT NULL,
+	text VARCHAR(256) NOT NULL,
+	is_correct BOOLEAN NOT NULL,
+	PRIMARY KEY (ans_id)
+);
+
+CREATE TABLE quiz (
+	quiz_id INT NOT NULL,
+	name VARCHAR(128) NOT NULL,
+	maxTime int NOT NULL,
+	PRIMARY KEY (quiz_id)
+);
+
+CREATE TABLE user (
+	user_id INT NOT NULL,
+	username VARCHAR(64) NOT NULL,
+	password BINARY NOT NULL,
+	user_type CHAR NOT NULL,
+	PRIMARY KEY (user_id),
+	UNIQUE (username)
+);
+
+CREATE TABLE quest_quiz (
+	quest_id INT NOT NULL,
+	quiz_id INT NOT NULL,
+	CONSTRAINT quest_guiz_pk PRIMARY KEY (quest_id, quiz_id)
```

Comment:
Lepiej wprowadzić sztuczne ID dlatego, że to co napisałaś jest bardzo niewygodne w implementacji, ja bym zrobił tak:
`CREATE TABLE quest_quiz (
id INT NOT NULL,
question_id INT NOT NULL REFERENCES question(id),
quiz_id INT NOT NULL REFERENCES quiz(id),
CONSTRAINT quest_quiz_pkey PRIMARY KEY (id),
CONSTRAINT u_question_id_quiz_id UNIQUE (question_id, quiz_id)
);`